docs/examples/straight_skeleton_2_holes.py

```diff
@@ -0,0 +1,42 @@
+from compas.datastructures import Graph
+from compas.geometry import Polygon
+from compas_viewer import Viewer
+
+from compas_cgal.straight_skeleton_2 import create_interior_straight_skeleton_with_holes
+
+points = [
+    (-1.91, 3.59, 0.0),
+    (-5.53, -5.22, 0.0),
+    (-0.39, -1.98, 0.0),
+    (2.98, -5.51, 0.0),
+    (4.83, -2.02, 0.0),
+    (9.70, -3.63, 0.0),
+    (12.23, 1.25, 0.0),
+    (3.42, 0.66, 0.0),
+    (2.92, 4.03, 0.0),
+    (-1.91, 3.59, 0.0),
+]
+
+holes = [
+    [(0.42, 0.88, 0.0), (1.1, -1.0, 0.0), (-1.97, -0.93, 0.0), (-1.25, 1.82, 0.0)],
+    [(4.25, -0.64, 0.0), (2.9, -3.03, 0.0), (2.12, -2.16, 0.0), (2.89, -0.36, 0.0)],
+    [(10.6, 0.29, 0.0), (9.48, -1.54, 0.0), (5.48, -1.26, 0.0), (5.98, -0.04, 0.0)],
+]
+
+
+polygon = Polygon(points)
+holes = [Polygon(hole) for hole in holes]
+lines = create_interior_straight_skeleton_with_holes(polygon, holes)
+graph = Graph.from_lines(lines)
```

Comment:
if the return value is a list polylines (not lines), it will not always be possible to directly convert this into a graph. perhaps it would be helpful to have a helper for making the conversion...

Reply:
The skeleton function is implemented to always return lines, so it should work to always convert them to a graph.

But I agree that the conversion is not proper yet. Ideally the return value should not be `PolylinesNumpy`, but a `compas.datastructures.mesh`, as it could represent CGAL's data type for the `StraightSkeleton_2`:
[https://doc.cgal.org/Manual/4.1/doc_html/cgal_manual/Straight_skeleton_2_ref/Concept_StraightSkeleton_2.html#Cross_link_anchor_1072](https://doc.cgal.org/Manual/4.1/doc_html/cgal_manual/Straight_skeleton_2_ref/Concept_StraightSkeleton_2.html#Cross_link_anchor_1072)

I anyway plan to add some other functions like the `create_interior_weighted_straight_skeleton_2`. So maybe I will tackle this in the next PR.

Reply:
ok so current one can be merged?

Reply:
yes